src/main/java/com/google/devtools/build/lib/analysis/test/TestRunnerAction.java

```diff
@@ -391,11 +391,10 @@ public boolean showsOutputUnconditionally() {
         builder.add(
             Pair.of(TestFileNameConstants.TEST_WARNINGS, resolvedPaths.getTestWarningsPath()));
       }
-      if (resolvedPaths.getUndeclaredOutputsZipPath().exists()) {
+      if (resolvedPaths.getUndeclaredOutputsDir().exists()) {
         builder.add(
             Pair.of(
-                TestFileNameConstants.UNDECLARED_OUTPUTS_ZIP,
```

Comment:
Not sure whether internal code relies on this name. Please keep an eye on the internal usages and tests when importing.

Reply:
Thanks for the heads up. The preexisting constant must stay. I've also made changes to handle the zipped/unzipped cases differently, as some internal code relies on the zip file (not the directory) being a spawn output.